### PR TITLE
Resolve UI and breaking changes from service

### LINKED ISSUE #4942: Adapt Lisk Service endpoint breaking changes
### Description

Adapt Lisk Service endpoint breaking changes

### Additional Information

Following are the GitHub issues addressed since the last tag:

- Update validateBLSKey endpoint [#1579](https://github.com/LiskHQ/lisk-service/issues/1578) (Breaking change)
  - Updates POST /validator/validateBLSKey -> POST /validator/validate-bls-key (RPC: post.validator.validate-bls-key)
- Extend schemas endpoint [#1571](https://github.com/LiskHQ/lisk-service/issues/1571)
  - Extends the response to include the ccm schema
- Update endpoints [#1592](https://github.com/LiskHQ/lisk-service/issues/1592) (Breaking change)
  - Updates GET /reward/inflation -> GET /reward/annual-inflation (RPC: get.reward.annual-inflation)
  - Updates GET /peers -> GET /network/peers (RPC: get.network.peers)
- Extend the blockchain apps endpoint [#1567](https://github.com/LiskHQ/lisk-service/issues/1567)
  - Extends the response to include the escrow information
- Incorrect total in response meta [#1563](https://github.com/LiskHQ/lisk-service/issues/1563)
  - Fixes bugs in the response meta for the following endpoints
  
- Fix padding issues on stakes summary page
image
image

- Fix ui issue on sign transaction from wallet connect

### PR DESCRIPTION
**Pull request by eniolam1000752**
### What was the problem?

This PR resolves #4942, #4962

### How was it solved?

- [x] Resolve breaking changes due to changes to endpoints
- [x] Fix ui issue on wallet connect transaction request summary
- [x] Fix issue on staking summary modal

### How was it tested?

- Visually

### CODE REVIEW

**Review by oskarleonard — DISMISSED** [review]

File: src/modules/transaction/components/TransactionDetails/sender.js [hunk 1]
```diff
@@ -10,7 +9,6 @@ import ValueAndLabel from './valueAndLabel';
 const Sender = ({ t }) => {
   const { activeToken, transaction, network } = React.useContext(TransactionDetailsContext);
   const validatorName = getValidatorName(transaction, activeToken);
-  const senderLabel = getModuleCommandSenderLabel()[transaction.moduleCommand];
```
Comment (ManuGowda) [inline src/modules/transaction/components/TransactionDetails/sender.js:13]: why was this removed?

**Review by ManuGowda — COMMENTED** [review]
rest of the code changes LGTM, one comment

File: src/modules/transaction/components/TransactionDetails/sender.js (hunk 1 above)
Comment (eniolam1000752) [reply src/modules/transaction/components/TransactionDetails/sender.js:13]: Already for the wallet connect its would only be a constant that the label would be `Sender`. having  to use `getModuleCommandSenderLabel` to get the summary title for sender based on tx module would be on over kill I think.

**Review by ManuGowda — APPROVED** [review]

**Review by oskarleonard — APPROVED** [review]

File: src/modules/transaction/components/TransactionDetails/sender.js (hunk 1 above)
Comment (eniolam1000752) [reply src/modules/transaction/components/TransactionDetails/sender.js:13]: just taking a second look and yhea you are right

**Pull request merged**